### DISCUSSION TITLE
Printing 2 line of text next to a QR code

### DISCUSSION THREAD

**Discussion by Marco375** [igorocampos/ESCPOS#35] [Q&A] [answered]
Hi,

How would I use this library to print 2 lines of text next to a QR code?

**Comment by igorocampos** [comment, marked as answer]
Hi @Marco375, unfortunately, I did not implement this feature in my library. If you are willing to contribute this could perhaps be accomplished using the following ESC POS command:
![image](https://user-images.githubusercontent.com/10572656/220979360-24f06ff3-7765-41ed-aa88-80f3443a35dd.png)


If not, then I know one could achieve this goal through page mode and its various commands, which would require a lot more implementations, but it's doable.
